engines/python/src/main/java/ai/djl/python/engine/PyProcess.java

```diff
@@ -132,12 +158,16 @@ Output predict(Input inputs, int timeout, boolean initialLoad) {
     }
 
     synchronized void startPythonProcess() {
+        // Do not start python process in worker nodes
+        if (Utils.getenv("WORKER_INDEX") != null) {
```

Comment:
I don't think we need this. DJLServing should not start on worker

Reply:
We need to start DJLServing on worker, otherwise it will container djl_python_engine.py not found.

Reply:
Removed.